app.py

```diff
@@ -72,37 +95,27 @@ def generate_response(prompt):
     completion_tokens = completion.usage.completion_tokens
     return response, total_tokens, prompt_tokens, completion_tokens
 
+# Define a custom style for the container
+container_style = 'position: relative; top: 300px; left: 100px;'
+
 # container for chat history
 response_container = st.container()
 # container for text box
 container = st.container()
+container.markdown('<div style="{}">'.format(container_style), unsafe_allow_html=True)
 
 with container:
     with st.form(key='my_form', clear_on_submit=True):
-        user_input = st.text_area("You:", key='input', height=100)
-        submit_button = st.form_submit_button(label='Send')
+        user_input = st.text_area("", placeholder="What do you want to learn today?", key='input', height=10)
+        submit_button = st.form_submit_button(label= '⏩')
 
     if submit_button and user_input:
         output, total_tokens, prompt_tokens, completion_tokens = generate_response(user_input)
         st.session_state['past'].append(user_input)
         st.session_state['generated'].append(output)
-        st.session_state['model_name'].append(model_name)
-        st.session_state['total_tokens'].append(total_tokens)
-
-        # from https://openai.com/pricing#language-models
-        if model_name == "GPT-3.5":
-            cost = total_tokens * 0.002 / 1000
-        else:
-            cost = (prompt_tokens * 0.03 + completion_tokens * 0.06) / 1000
-
-        st.session_state['cost'].append(cost)
-        st.session_state['total_cost'] += cost
 
 if st.session_state['generated']:
     with response_container:
         for i in range(len(st.session_state['generated'])):
-            st.markdown(f"**You**: {st.session_state['past'][i]}", unsafe_allow_html=True)
-            st.markdown(f"**waam**: {st.session_state['generated'][i]}", unsafe_allow_html=True)
-            st.write(
-                f"Model used: {st.session_state['model_name'][i]}; Number of tokens: {st.session_state['total_tokens'][i]}; Cost: ${st.session_state['cost'][i]:.5f}")
-            counter_placeholder.write(f"Total cost of this conversation: ${st.session_state['total_cost']:.5f}")
+            message(st.session_state["past"][i], is_user=True, key=str(i) + '_user')
+            message(st.session_state["generated"][i], key=str(i))
```

Comment:
I think wee should keep these two lines as they were as st.markdown.

Good idea deleting the last three lines about tokens though.